src/App.tsx

```diff
@@ -1,24 +1,288 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { Error } from './types/Error';
+import {
+  getTodos,
+  deleteTodo,
+  createTodo,
+  updateTodo,
+} from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
 import { UserWarning } from './UserWarning';
+import { TodoError } from './components/TodoError/TodoError';
+import { TodoLoadingItem } from './components/TodoLoadingItem/TodoLoadingItem';
+import {
+  filterTodos,
+  getItemsLeftCountMessage,
+  getUpdatedTodos,
+} from './utils/functions';
+import { client } from './utils/fetchClient';
 
-const USER_ID = 0;
+const USER_ID = 11521;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<Status>(Status.All);
+  const [errorMessage, setErrorMessage] = useState(Error.None);
+  const [value, setValue] = useState('');
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [areSubmiting, setAreSubmiting] = useState(false);
+  const [wasEdited, setWasEdited] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [processingIds, setProcessingIds] = useState<number[]>([]);
+  const [togglingId, setTogglingId] = useState<number | null>(null);
+
+  const inputRef = useRef<HTMLInputElement | null>(null);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => setErrorMessage(Error.Load));
+  }, []);
+
+  useEffect(() => {
+    if (errorMessage) {
+      setTimeout(() => {
+        setErrorMessage(Error.None);
+      }, 3000);
+    }
+  }, [errorMessage]);
```

Comment:
https://upmostly.com/tutorials/settimeout-in-react-components-using-hooks

Reply:
Please read the article, there is an example of how to correctly work with timers in `useEffect`.